### ISSUE TITLE
rec模型微调hard case时，lr要设置为多少？

### ISSUE THREAD

**Issue by nissansz** [PaddlePaddle/PaddleOCR#12054]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

rec模型微调hard case时，lr要设置为多少？
默认配置如下
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0005
  regularizer:
    name: L2

**Comment by UserWangZz** [comment]
你可以从数据集中选一部分子集做一组实验，探究模型性能和学习率的关系，然后使用效果最好的学习率在全数据集上进行训练

**Comment by nissansz** [comment]
子数据集要多少张图片

**Comment by UserWangZz** [comment]
要是想快速探究可以选少一些，要是想效果好可以尝试多一些数据，尽量全部场景都覆盖到

**Comment by yilong2001** [comment]
{
"img": "202312070001_crop_13.jpg",
"gen": [
"送检医师：柏维昌",
0.9480968117713928
],
"label": "送检医师：柏继昌"
}
您好，对于微调后的模型，预测错误的 case，如何增加数据、再进行微调效果更好？
比如：202312070001_crop_13.jpg 做一些数据增广，还是识别错误的内容：“柏继昌” ，使用更多的真实数据？

**Comment by UserWangZz** [comment]
一方面尽可能多增加真实数据，还可以根据已有数据，进行数据生成，可以参考文档
https://github.com/PaddlePaddle/PaddleOCR/blob/main/StyleText/README_ch.md
